.internal-ci/docker/Dockerfile.full-service-with-build

```diff
@@ -59,15 +59,15 @@ RUN  --mount=type=cache,target=/opt/cargo/git \
   --mount=type=cache,target=/opt/cargo/registry/index \
   --mount=type=cache,target=/opt/cargo/registry/cache \
   --mount=type=cache,target=/app/target \
-  cargo build --locked --release ${BUILD_OPTS}
-
-RUN cp /app/target/release/full-service /usr/local/bin/
-RUN cp /app/target/release/signer /usr/local/bin/
-RUN cp /app/target/release/signer-service /usr/local/bin/
-RUN cp /app/target/release/validator-service /usr/local/bin/
-RUN cp /app/target/release/wallet-service-mirror-private /usr/local/bin/
-RUN cp /app/target/release/wallet-service-mirror-public /usr/local/bin/
-RUN cp /app/target/release/generate-rsa-keypair /usr/local/bin/
+  cargo build --locked --release ${BUILD_OPTS} \
+  && cp /app/target/release/full-service \ 
```

Comment:
I didn't know you could do multiple sources with cp
> Copy SOURCE to DEST, or multiple SOURCE(s) to DIRECTORY.

Reply:
Thanks for the prompt, after-hours review!

and yeah, cp letting you have an arbitrary number of sources is particularly useful when combined with the shell's brace expansion pattern.  I didn't use it here for readability, but you can do something like:

cp /long/path/i/dont/want/type/over/and/over/{a,b,c}.txt /dest/path/thats/also/long

to copy a.txt, b.txt, and c.txt to a new place.

works with mv too.